frontend/src/metabase-types/guards/click-behavior.ts

```diff
@@ -0,0 +1,77 @@
+import type {
+  BaseActionClickBehavior,
+  DeleteActionClickBehavior,
+  ImplicitActionClickBehavior,
+  InsertActionClickBehavior,
+  UpdateActionClickBehavior,
+} from "metabase-types/api";
+
+const isRecord = (
+  value: unknown,
+): value is Record<string | number | symbol, unknown> => {
+  return typeof value === "object" && value !== null;
```

Comment:
I speculate here, but `[]` will also pass this check 😄

Reply:
Renamed to `isObject`: 8ca61c2de5c222ebd758e47c44cbc1cc9d5a4ed5